WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.m

```diff
@@ -369,12 +369,16 @@ - (BOOL)fb_dismissKeyboardWithKeyNames:(nullable NSArray<NSString *> *)keyNames
     if (nil != auditTypeValue) {
       auditType = valuesToNamesMap[auditTypeValue] ?: [auditTypeValue stringValue];
     }
+    id<FBXCElementSnapshot> snapshot = [extractIssueProperty(issue, @"element") fb_takeSnapshot];
+    NSDictionary *elementObject = snapshot ? [self.class dictionaryForElement:snapshot recursive:NO] : @{};
```

Comment:
is that really what is needed? Just to get all attributes of an element?
I though element path in the tree is necessary

Reply:
The idea behind my proposal is to provide all the attributes of the element for which the problem was found, so that it would be similar to the native AccessibilityAudit API.
This will be enough for writing automated tests and debugging and much more.
In case I’m missing something could you point me to example of getting element path in appium

Reply:
it is ok, just wanted to confirm it is exactly what is needed. Then I assume `elementAttributes` would be a proper name

Reply:
Ok, I got it.
Done 👌